documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -1,50 +1,56 @@
 // Module included in the following assemblies:
 //
-// deploying/assembly_deploy-verify.adoc
-// getting-started.adoc
+// deploying/assembly-securing-access.adoc
 
 [id='setup-external-clients-{context}']
-= Setting up client access to a Kafka cluster using listeners
+= Example: Setting up secure client access
 
 [role="_abstract"]
-Using the address of the Kafka cluster, you can provide access to a client in the same Kubernetes cluster; or provide external access to a client on a different Kubernetes namespace or outside Kubernetes entirely.
 This procedure shows how to configure client access to a Kafka cluster from outside Kubernetes or from another Kubernetes cluster.
+It's split into two parts:
 
-A Kafka listener provides access to the Kafka cluster.
-Client access is secured using the following configuration:
+* Securing Kafka brokers
+* Securing user access to Kafka
 
-. An external listener is configured for the Kafka cluster, with TLS encryption and mTLS authentication, and Kafka `simple` authorization enabled.
-. A `KafkaUser` is created for the client, with mTLS authentication, and Access Control Lists (ACLs) defined for `simple` authorization.
+.Resource configuration
 
-You can configure your listener to use mutual `tls`, `scram-sha-512`, or `oauth` authentication.
-mTLS always uses encryption, but encryption is also recommended when using SCRAM-SHA-512 and OAuth 2.0 authentication.
+Client access to the Kafka cluster is secured with the following configuration:
 
-You can configure `simple`, `oauth`, `opa`, or `custom` authorization for Kafka brokers.
-When enabled, authorization is applied to all enabled listeners.
+. An external listener is configured with TLS encryption and mutual TLS (mTLS) authentication in the `Kafka` resource, as well as `simple` authorization.
+. A `KafkaUser` is created for the client, utilizing mTLS authentication, and Access Control Lists (ACLs) are defined for `simple` authorization.
+
+At least one listener supporting the desired authentication must be configured for the `KafkaUser`.
+
+To ensure compatibility between Kafka brokers and clients, configuration of authentication and authorization mechanisms must align:
 
-When you configure the `KafkaUser` authentication and authorization mechanisms, ensure they match the equivalent Kafka configuration:
+* `Kafka.spec.kafka.listeners[*].authentication` must match `KafkaUser.spec.authentication`
```

Comment:
This applies only for supported types. Not for custom or OAuth for example.

Reply:
Yeah. I've changed the order and updated as follows:

To ensure compatibility between Kafka brokers and clients, configuration of the following authentication and authorization mechanisms must align:

* For `type: tls` and `type: scram-sha-512` authentication types, `Kafka.spec.kafka.listeners[*].authentication` must match `KafkaUser.spec.authentication`
* For `type: simple` authorization,`Kafka.spec.kafka.authorization` must match `KafkaUser.spec.authorization`

For example, mTLS authentication for a user is only possible if it's also enabled in the Kafka configuration.